lib/debug/session.rb

```diff
@@ -935,7 +935,7 @@ def event type, *args
       case type
       when :suspend_bp
         bp, i = *args
```

Comment:
I think it's because the local assignments are wrong. it should be this instead:

```suggestion
        i, bp = *args
```

the current message is actually correct.

Reply:
the caller:

https://github.com/ruby/debug/blob/bbf475f976703788e6622b682b5a58d68297395a/lib/debug/session.rb#L108

Reply:
You are right, Thanks!